lms/templates/courseware/course_navigation-sneak-peek-warning.html

```diff
@@ -14,10 +14,12 @@
   else:
     reg_url = reverse('register_user')
   %>
-   ${_("Non-registered mode.  {tag_start}Register{tag_end} to save your course progress.")\
+   ${_("Non-registered mode.  {tag_start}Register{tag_end} to save your course progress. By exploring the course, you are agreeing to our {tos_link_start}Terms of Service{tag_end} and {privacy_link_start}Privacy Policy{tag_end}. Please read them carefully.")\
```

Comment:
I'm not really following this comment as since this is all this file does it is effectively in an external file. I must be missing something.